orion/pkg/tests_integration/session_integration_test.go

```diff
@@ -0,0 +1,183 @@
+func Test_CreateSessions(t *testing.T) {
+	resetTable(t, domains.TABLE_SESSIONS)
+	resetTable(t, domains.TABLE_CLASSES)
+	resetTable(t, domains.TABLE_SEMESTERS)
+	resetTable(t, domains.TABLE_LOCATIONS)
+	resetTable(t, domains.TABLE_PROGRAMS)
+
+	// Create
+	start := time.Now().UTC()
+	mid := start.Add(time.Minute * 30)
+	end := start.Add(time.Hour)
+	prog1 := createProgram("fast_track", "Fast Track", 1, 12, "descript1")
+	prog2 := createProgram("slow_track", "Slow Track", 1, 12, "descript1")
+	loc1 := createLocation("loc_1", "4040 Location Rd", "City", "MA", "77294", "Room 1")
+	semester1 := createSemester("2020_spring", "Spring 2020")
+	semester2 := createSemester("2020_fall", "Fall 2020")
+	class1 := createClass("fast_track", "2020_spring", "class_A", "loc_1", "5 pm - 7 pm", start, end)
+	class2 := createClass("slow_track", "2020_fall", "class_B", "loc_1", "3 pm - 7 pm", start, end)
+	session1 := createSession("fast_track_2020_spring_class_A", mid, end, false, "special lecture from guest")
+	session2 := createSession("fast_track_2020_spring_class_A", start, end, true, "May 5th regular meeting")
+	session3 := createSession("slow_track_2020_fall_class_B", start, end, false, "May 5th regular meeting")
+	body1 := createJsonBody(prog1)
+	body2 := createJsonBody(prog2)
+	body3 := createJsonBody(loc1)
+	body4 := createJsonBody(semester1)
+	body5 := createJsonBody(semester2)
+	body6 := createJsonBody(class1)
+	body7 := createJsonBody(class2)
+	body8 := createJsonBody(session1)
+	body9 := createJsonBody(session2)
+	body10 := createJsonBody(session3)
+	recorder1 := sendHttpRequest(t, http.MethodPost, "/api/programs/v1/create", body1)
+	recorder2 := sendHttpRequest(t, http.MethodPost, "/api/programs/v1/create", body2)
+	recorder3 := sendHttpRequest(t, http.MethodPost, "/api/locations/v1/create", body3)
+	recorder4 := sendHttpRequest(t, http.MethodPost, "/api/semesters/v1/create", body4)
+	recorder5 := sendHttpRequest(t, http.MethodPost, "/api/semesters/v1/create", body5)
+	recorder6 := sendHttpRequest(t, http.MethodPost, "/api/classes/v1/create", body6)
+	recorder7 := sendHttpRequest(t, http.MethodPost, "/api/classes/v1/create", body7)
+	recorder8 := sendHttpRequest(t, http.MethodPost, "/api/sessions/v1/create", body8)
+	recorder9 := sendHttpRequest(t, http.MethodPost, "/api/sessions/v1/create", body9)
+	recorder10 := sendHttpRequest(t, http.MethodPost, "/api/sessions/v1/create", body10)
+	assert.EqualValues(t, http.StatusOK, recorder1.Code)
+	assert.EqualValues(t, http.StatusOK, recorder2.Code)
+	assert.EqualValues(t, http.StatusOK, recorder3.Code)
+	assert.EqualValues(t, http.StatusOK, recorder4.Code)
+	assert.EqualValues(t, http.StatusOK, recorder5.Code)
+	assert.EqualValues(t, http.StatusOK, recorder6.Code)
+	assert.EqualValues(t, http.StatusOK, recorder7.Code)
+	assert.EqualValues(t, http.StatusOK, recorder8.Code)
+	assert.EqualValues(t, http.StatusOK, recorder9.Code)
+	assert.EqualValues(t, http.StatusOK, recorder10.Code)
```

Comment:
lol I guess all of this from line 24 is kind of necessary. I wonder if we can make it better